core/translations/cs.json

```diff
@@ -611,6 +611,8 @@
     "recovery__title_recover": "Obnovit peněženku",
     "recovery__title_remaining_shares": "Zbývající části",
     "recovery__type_word_x_of_y_template": "Zadejte slovo {0} z {1}",
+    "recovery__unlock_repeated_backup": "Vytvořit další zálohu?",
+    "recovery__unlock_repeated_backup_verb": "Odemknout zálohu",
```

Comment:
do we still use "unlock backup"?  We wanted to change it to "Create additional backup?"

Reply:
This is the button. Here is how it looks...

![image](https://github.com/trezor/trezor-firmware/assets/111120/6e95f54b-e5e8-4e82-ba22-b3f3dde907cb)

Reply:
Ok, we should change it to `Create backup` on the button. Can we do it?

Reply:
Speaking of which, it does indeed not match your screenshot. Good catch. Let me change it now.